src/Core/Framework/DependencyInjection/store.xml

```diff
@@ -47,8 +47,7 @@
 
         <service id="Shopware\Core\Framework\Store\Services\OpenSSLVerifier">
             <argument type="collection">
-                <argument>%kernel.project_dir%/vendor/shopware/platform/src/Core/Framework/Store/public.key</argument>
-                <argument>%kernel.project_dir%/vendor/shopware/core/Framework/Store/public.key</argument>
+                <argument>%framework.bundle_dir%/Store/public.key</argument>
```

Comment:
should we maybe prefix this? `framework.bundle_dir` sounds quite generic and not Shopware specific

Reply:
I suggest, we prefix it with `bundle.`. This would still not be specific to Shopware, but in the same way the `Framework` bundle is also not named very Shopware specific. Would this be ok?

Reply:
`bundle` is already part of the name. why not adding `shopware.` as prefix? we have many container parameter names starting with this

Reply:
Because this will affect every bundle that extends `\Shopware\Core\Framework\Bundle`, so every plugin will also have `shopware.` for prefix. Is this ok?

Reply:
oh yes, you are right :see_no_evil: 
no, that would be not so cool, I guess.

then probably the core bundles need some re-naming, but that should not be part of this PR :blush: 
then keep it, as it is :+1:

Reply:
Then I assume, my work here is done.

Reply:
Also: What!? You are planning to rename the core bundles? This will break so much BC 😭

Reply:
![grafik](https://user-images.githubusercontent.com/1133593/92398802-7f4e1c00-f129-11ea-9330-3a53433798ee.png)

Reply:
never did I speak about "planning"... 
this was just a wild guess to make the core bundles more recognisable :blush: and as you already find out, this won't happen because of reasons :wink: